cmd/utils/dev.go

```diff
@@ -175,7 +175,7 @@ func GetDevFromManifest(manifest *model.Manifest, devName string) (*model.Dev, e
 		return nil, ErrNoDevSelected
 	}
 
-	options := []string{}
+	var options []string
```

Comment:
✨ fix suggested by the linter [[1]](https://deepsource.io/gh/okteto/okteto/run/851a6b0a-999a-4ffb-89ed-cd9c008f703b/go/GO-W1027?listindex=0)